test-fixture/integration-reactive/src/main/kotlin/com/linecorp/kotlinjdsl/test/reactive/querydsl/predicate/AbstractCriteriaQueryDslPredicateIntegrationTest.kt

```diff
@@ -302,4 +305,57 @@ abstract class AbstractCriteriaQueryDslPredicateIntegrationTest<S> : CriteriaQue
         // then
         assertThat(orderItemIds).isEmpty()
     }
+
+    @Test
+    fun `exists - subquery`() = runBlocking {
+        val existFoundOrders = withFactory { queryFactory ->
+            queryFactory.listQuery<Order> {
```

Comment:
It seems that explicit generic type specification in this code is unnecessary. If there are no compilation issues, please remove them. The reason why the generic type specification was necessary in the previous code was the situation when the Kotlin version was low.

Reply:
All the test has unnecessary generic too, so can I remove all??

Reply:
sure. thank you.